### DISCUSSION TITLE
Redlining - Konstruktions-Kontext-Menü erscheint nicht

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#67] [Q&A] [answered]
Wahrscheinlich wieder ein doofer Konfigurationsfehler, aber ich komme nicht drauf was es ist.
Vorgangsweise: Zeichnen (Redlining) -> Fläche -> Ersten Punkt setzen -> Rechte Maustaste klicken
So sieht es im Testsystem aus:
![kontext_test](https://user-images.githubusercontent.com/100088983/235890409-972ed7f4-336b-4703-a100-6caa91479be0.png)

Und so sieht es im Produktivsystem aus:
![kontext_atlas](https://user-images.githubusercontent.com/100088983/235890494-acd7b5d8-e744-4c24-af82-0f0696196b68.png)

Hat jemand einen Lösungsvorschlag für das Problem?

**Comment by gis-eni** [comment]
Browser cache leeren?

**Reply by faulgurke** [reply]
Leider nein - alles gelöscht - gleiches Ergebnis.

**Comment by gis-eni** [comment]
Kann man sich das irgendwo anschauen?

**Comment by gis-eni** [comment]
Steht bei dir in der ``portal.config`` irgendwo:

``` xml
<add key="use-custom-recommendation-js" value="true" />
```

**Comment by faulgurke** [comment, marked as answer]
Der Fehler ist dank Hilfe von Jürgen G. entdeckt und behoben.
custom.js: webgis.usability.sketchContextMenu war auf false (so geht's natürlich nicht).

![custom_js](https://user-images.githubusercontent.com/100088983/236387231-8cc48786-f069-4a29-84a3-5ad17cddbf4a.png)
